BusinessLayerTests/InventoryCsvManagerTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Models;
+using Moq;
+using Serilog;
+
+namespace BusinessLayer.Tests
+{
+    [TestClass]
+    public class InventoryCsvManagerTests
+    {
+        private InventoryCsvManager _inventoryCsvManager;
+        private IConfigurationRoot _mockedConfig;
+
+        private List<Contact> contacts;
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            var configMock = new Mock<IConfigurationRoot>();
+            configMock.SetupGet(x => x["INVENTORY_PATH"])
+                .Returns(Path.Combine(Directory.GetCurrentDirectory(), "inventory.csv"));
+            _mockedConfig = configMock.Object;
+            var mockLog = new Mock<ILogger>().Object;
```

Comment:
be consistent in your naming => logMock

Reply:
Renamed into mockedLogger (Because it's the Object and not the Mock itself)